src/library_analyzer/processing/api/purity_analysis/model/_purity.py

```diff
@@ -356,6 +392,37 @@ class Unknown(ImpurityReason, ABC):
     """Superclass for unknown type impurity reasons."""
 
 
+@dataclass
+class UnknownProto(Unknown):
+    """Class for UnknownCalls which are not fully determined.
+
+    Attributes
+    ----------
+    symbol : Symbol | Reference
```

Comment:
```suggestion
    symbol:
```

Types are already in the code. No need to change it everywhere, but it's not needed for new code or code close to changed code.

Reply:
Since we talked about that on multiple occasions, the next commit removes all type annotations from all the docstrings.